python/Ganga/GPIDev/Lib/Config/Config.py

```diff
@@ -266,10 +266,14 @@ def config_file_as_text():
                                 def_value = "\n# ".join(def_lines)
                         except AttributeError as err:
                             pass
+                    print 'o: ', o, ' def_value: ', def_value, ' value: ', value
```

Comment:
Did realise this was a print (and not a logging) statement. It was quite helpful when running that this was printed out (it is actually half the way towards the system that we discussed for moving values forward). Should probably just be deleted for now.